### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by raisavarghese**
Added steps for integration and deploy of rhtap

### CODE REVIEW

**Comment by raisavarghese** [comment]
@jkopriva

**Comment by jkopriva** [comment]
@raisavarghese Hi looks good!

File: README.md [hunk 1]
```diff
@@ -100,3 +100,30 @@ Please refer to the [CONTRIBUTING.md](CONTRIBUTING.md) file for more information
 
 
 [helm]: https://helm.sh/
+
+# Deploy RHTAP
```
Comment (otaviof) [inline README.md]: Could you make this a `.h2` header, to follow the rest of the document?

File: README.md [hunk 2]
```diff
@@ -100,3 +100,30 @@ Please refer to the [CONTRIBUTING.md](CONTRIBUTING.md) file for more information
...
+Follow the below steps to deploy RHTAP on Openshift cluster.
+
+1.Clone the repositry.Run the command 'make` from the rhtap-cli directory.
```
Comment (otaviof) [inline README.md]: Could you add spaces after the `.` (dots), please? All the list is lacking space after dot/period, please consider.

File: README.md [hunk 3]
```diff
@@ -100,3 +100,30 @@ Please refer to the [CONTRIBUTING.md](CONTRIBUTING.md) file for more information
...
+This will create a bin directory.
+
+2.Edit the config.yaml file for select or deselect the components from installation.
```
Comment (otaviof) [inline README.md]: Could you create a link to the repository file `config.yaml`? There are examples before that you can copy and paste.

File: README.md [hunk 4]
```diff
@@ -100,3 +100,30 @@ Please refer to the [CONTRIBUTING.md](CONTRIBUTING.md) file for more information
...
+
+  Eg : Change the lines as below to disable installation of components ACS and Quay
+
+  `redHatAdvancedClusterSecurity:
+      enabled: false`
+      
+  `redHatQuay:
+      enabled: false`
```
Comment (otaviof) [inline README.md]: This block should become a [code block](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks).

File: README.md [hunk 5]
```diff
@@ -100,3 +100,30 @@ Please refer to the [CONTRIBUTING.md](CONTRIBUTING.md) file for more information
...
+      
+3.Run the command `bin/rhtap-cli` to display help text that shows all the supported commands and options
+
+4.Run the command `bin/rhtap-cli integration` to provide integrations to external components
+  
+  `bin/rhtap-cli integration --help` will list the options supported
+  
+5.Finally run the command `bin/rhtap-cli deploy` to proceed with RHTAP deployment
```
Comment (otaviof) [inline README.md]: Please simply refer to `rhtap-cli` instead of `bin/rhtap-cli`.

File: README.md [hunk 6]
```diff
@@ -100,3 +100,30 @@ Please refer to the [CONTRIBUTING.md](CONTRIBUTING.md) file for more information
...
+      enabled: false`
+      
+  `redHatQuay:
+      enabled: false`
+      
+3.Run the command `bin/rhtap-cli` to display help text that shows all the supported commands and options
+
+4.Run the command `bin/rhtap-cli integration` to provide integrations to external components
+  
+  `bin/rhtap-cli integration --help` will list the options supported
```
Comment (otaviof) [inline README.md]: Another place where it should become a code block, but this time Bash formatted.

**Review by otaviof — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (raisavarghese) [reply README.md]: Done with all changes

File: README.md [hunk 7]
```diff
@@ -100,3 +100,39 @@ Please refer to the [CONTRIBUTING.md](CONTRIBUTING.md) file for more information
...
+2.Edit the [`config.yaml`](config.yaml) file for select or deselect the components from installation. 
+
+  Eg : Change the lines as below to disable installation of components ACS and Quay 
+
+  ```yaml
+  redHatAdvancedClusterSecurity: 
+   enabled: false 
+      
+  redHatQuay: 
+   enabled: false
```
Comment (otaviof) [inline README.md]: Please unindent this block.

**Review by otaviof — CHANGES_REQUESTED** [review]
Just a final change, but thanks for the docs!

**Review by otaviof — APPROVED** [review]

**Pull request merged**